subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -36,6 +36,7 @@ enum flags {
 	FLAG_CTRL_SRV_MANUALLY_ENABLED,
 	FLAG_STARTED,
 	FLAG_RESUME_TIMER,
+	FLAG_SETTINGS_SET,
```

Comment:
Nitpick:
```suggestion
	FLAG_SETTINGS_LOADED,
```
Just to get a word that is clearly past tense, as `SET` could be both an imperative and a status